apps/wallet/src/ui/index.tsx

```diff
@@ -47,9 +48,13 @@ function renderApp() {
                 <Provider store={store}>
                     <IntlProvider locale={navigator.language}>
                         <QueryClientProvider client={queryClient}>
-                            <ErrorBoundary>
-                                <App />
-                            </ErrorBoundary>
+                            <RpcClientContext.Provider
+                                value={api.instance.fullNode}
```

Comment:
Does anything make this value change? Curious if this will break if folks change networks?

Reply:
Bleh you're totally correct, I overlooked that `api.instance.fullNode` changes in value and isn't reactive. Let me fix this

Reply:
Okay I don't particularly like this solution, but I think this is the most straightforward path forward without re-wiring a bunch of Redux thunks and propagating more context instances around the codebase. Good catch on this!